docs/node-operators/get-started/setup-server.md

```diff
@@ -30,9 +30,9 @@ For production use, we recommend using the Linux binary on Ubuntu as this is the
 See the [infrastructure requirements](../requirements/infrastructure.md) page for a full list of what you need to run various parts of the Vega toolchain.
 
 ## Use the pre-built binaries
-One way to set up a Vega node is to use the pre-built binaries. 
+One way to set up a Vega node is to use the pre-built binaries. You'll use: Vega, Vega Wallet, Visor (optional - for easier protocol upgrades), and data node (optional).
```

Comment:
Just a note - Vega, Vega Wallet and data node are all subcommands of a vega binary.